### DISCUSSION TITLE
Review a fork

### DISCUSSION THREAD

**Discussion by pavelantik1** [swoosh/swoosh#786] [Q&A]
Hi.  I wanna add  AMP mail supporting for SMTP adapter. The update is ready and merged [to my fork's](https://github.com/pavelantik1/swoosh) main branch.
And what's next step? How long wait for the review?

**Comment by princemaple** [comment]
Hi,

Doing work on a fork won't notify the original repo maintainers. If you wanted to contribute your work back to the original project, you should send a pull request to the original repo, in this case swoosh.

It is a good idea to first discuss the implementation of a major new feature, though. In this case, AMP is not something I would like to support in the swoosh project directly. Hence, this feature will not be merged in, if a pull request is sent. However, you can extract the adapter implemention into a standalone repo, and use it as a git dependency, or publish it as a companion library and use it in your project directly.

Cheers.
